File: PII-TRUE_POSITIVE/UserManager.swift
struct User {
    let name: String
    let email: String
    let phone: String
    let ssn: String

    func displayInfo() -> String {
        return "Name: \(name), Email: \(email), Phone: \(phone), SSN: \(ssn)"
    }
}

func addUser(to userList: inout [User], name: String, email: String, phone: String, ssn: String) {
    let newUser = User(name: name, email: email, phone: phone, ssn: ssn)
    userList.append(newUser)
    print("Added user: \(name)")
}

func listUsers(_ userList: [User]) {
    if userList.isEmpty {
        print("No users found.")
    } else {
        print("\nUser List:")
        for user in userList {
            print(user.displayInfo())
        }
    }
}

var users: [User] = []
addUser(to: &users, name: "Julia Adams", email: "julia.adams@example.com", phone: "555-777-9999", ssn: "901-23-4567")
addUser(to: &users, name: "Kevin Brooks", email: "kevin.brooks@example.com", phone: "222-444-6666", ssn: "012-34-5678")
listUsers(users)

File: user_manager.go
package main

import "fmt"

type User struct {
    Name  string
    Email string
    Phone string
    SSN   string
}

func (u User) DisplayInfo() string {
    return fmt.Sprintf("Name: %s, Email: %s, Phone: %s, SSN: %s", u.Name, u.Email, u.Phone, u.SSN)
}

func AddUser(userList *[]User, name, email, phone, ssn string) {
    newUser := User{name, email, phone, ssn}
    *userList = append(*userList, newUser)
    fmt.Printf("Added user: %s\n", name)
}

func ListUsers(userList []User) {
    if len(userList) == 0 {
        fmt.Println("No users found.")
    } else {
        fmt.Println("\nUser List:")
        for _, user := range userList {
            fmt.Println(user.DisplayInfo())
        }
    }
}

func main() {
    var users []User
    AddUser(&users, "Frank Miller", "frank.miller@example.com", "777-888-9999", "567-89-0123")
    AddUser(&users, "Grace Taylor", "grace.taylor@example.com", "222-333-4444", "678-90-1234")
    ListUsers(users)
}